modules/orchestrator/proc-deploying-workflows-on-a-cluster.adoc

```diff
@@ -0,0 +1,108 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="proc-deploying-workflows-on-a-cluster.adoc_{context}"]
+= Deploying workflows on a cluster
+
+You can now deploy the workflow on a cluster, since the image has been pushed to the image registry and the deployment manifests have been made available.
+
+.Prerequisites
+
+* You have an {ocp-short} cluster with the following versions of components installed:
+
+*   {product} ({product-very-short}) `v1.5`
```

Comment:
I would align versions with 1.7 - so plugins v1.7.1 and serverless and serverless logic should be 1.36